scripts/cleanbox.sh

```diff
@@ -100,13 +108,7 @@ then
     echo "deb http://old-releases.ubuntu.com/ubuntu/ quantal-updates main restricted universe" > /etc/apt/sources.list
   fi
 
-  # Replace repos by archive for Debian Squeeze
-  grep -e "^6\." /etc/debian_version > /dev/null
-  squeeze=$?
-  if [ $squeeze -eq 0 ] ;
-  then
-    echo "deb http://archive.debian.org/debian/ squeeze main" > /etc/apt/sources.list
-  fi
+  apt-get update
```

Comment:
The update is not needed here, just within the if
I should have said just before the "fi"